eZ/Publish/Core/Search/Legacy/Content/WordIndexer/Repository/SearchIndex.php

```diff
@@ -84,13 +84,11 @@ public function addWords(array $words): void
             ->values(
                 [
                     'word' => ':word',
-                    'object_count' => ':one',
                 ]
-            )
-            ->setParameter(':one', 1, ParameterType::INTEGER);
+            );
 
         foreach ($words as $word) {
-            $query->setParameter('word', $word);
+            $query->setParameter(':word', $word);
```

Comment:
This is a mistake. Although not documented at all (actually PHPDoc examples are outdated and contradictory), [this line](https://github.com/doctrine/dbal/blob/2.13.0/lib/Doctrine/DBAL/Query/QueryBuilder.php#L1349) suggests that named parameters should not be prefixed when being passed to `setParameter`.

Reply:
So why not contribute to `doctrine/dbal` and open PR which fix usage examples ? :wink: